src/test/java/org/infinispan/persistence/mongodb/config/MongoDbStoreConfigurationSerializerTest.java

```diff
@@ -0,0 +1,8 @@
+package org.infinispan.persistence.mongodb.config;
+
+import org.infinispan.configuration.serializer.AbstractConfigurationSerializerTest;
+import org.testng.annotations.Test;
+
+@Test(testName = "org.infinispan.persistence.mongodb.config.MongoDbStoreConfigurationSerializerTest", groups = "functional")
+public class MongoDbStoreConfigurationSerializerTest extends AbstractConfigurationSerializerTest {
+}
```

Comment:
This class needs to override `compareStoreConfiguration()` like https://github.com/infinispan/infinispan/blob/main/persistence/rocksdb/src/test/java/org/infinispan/persistence/rocksdb/config/ConfigurationSerializerTest.java#L13